diff --git a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
index 23b936b..8042d01 100644
--- a/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
+++ b/node_modules/@strapi/admin/admin/src/components/LeftMenu/index.js
@@ -39,7 +39,7 @@ const LinkUser = styled(RouterNavLink)`

   &:hover {
     background: ${({ theme, logout }) =>
-      logout ? theme.colors.danger100 : theme.colors.primary100};
+  logout ? theme.colors.danger100 : theme.colors.primary100};
     text-decoration: none;
   }

@@ -50,13 +50,13 @@ const LinkUser = styled(RouterNavLink)`
   }
 `;

-const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
+const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks, initCondensedState, onToggle }) => {
   const buttonRef = useRef();
   const [userLinksVisible, setUserLinksVisible] = useState(false);
   const {
     logos: { menu },
   } = useConfigurations();
-  const [condensed, setCondensed] = usePersistentState('navbar-condensed', false);
+  const [condensed, setCondensed] = usePersistentState('navbar-condensed', initCondensedState);
   const { userDisplayName } = useAppInfos();
   const { formatMessage } = useIntl();

@@ -87,6 +87,13 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
     defaultMessage: 'Strapi Dashboard',
   });

+  const toggleSideBar = () => {
+    setCondensed((s) => {
+      onToggle(s)
+      return !s
+    })
+  }
+
   return (
     <MainNav condensed={condensed}>
       <NavBrand
@@ -201,16 +208,17 @@ const LeftMenu = ({ generalSectionLinks, pluginsSectionLinks }) => {
         </LinkUserWrapper>
       )}

-      <NavCondense onClick={() => setCondensed((s) => !s)}>
+      <NavCondense onClick={() => toggleSideBar()}>
+        <span>Zavřít</span>
         {condensed
           ? formatMessage({
-              id: 'app.components.LeftMenu.expand',
-              defaultMessage: 'Expand the navbar',
-            })
+            id: 'app.components.LeftMenu.expand',
+            defaultMessage: 'Expand the navbar',
+          })
           : formatMessage({
-              id: 'app.components.LeftMenu.collapse',
-              defaultMessage: 'Collapse the navbar',
-            })}
+            id: 'app.components.LeftMenu.collapse',
+            defaultMessage: 'Collapse the navbar',
+          })}
       </NavCondense>
     </MainNav>
   );
diff --git a/node_modules/@strapi/admin/admin/src/content-manager/pages/App/index.js b/node_modules/@strapi/admin/admin/src/content-manager/pages/App/index.js
index 9621b9d..3a6d68c 100644
--- a/node_modules/@strapi/admin/admin/src/content-manager/pages/App/index.js
+++ b/node_modules/@strapi/admin/admin/src/content-manager/pages/App/index.js
@@ -22,10 +22,11 @@ import NoPermissions from '../NoPermissions';
 import SingleTypeRecursivePath from '../SingleTypeRecursivePath';
 import LeftMenu from './LeftMenu';
 import useModels from './useModels';
+import {isMobile} from "./../../../pages/Admin";

 const cmPermissions = permissions.contentManager;

-const App = () => {
+const App = ({isLeftMenuOpen}) => {
   const contentTypeMatch = useRouteMatch(`/content-manager/:kind/:uid`);
   const { status, collectionTypeLinks, singleTypeLinks, models, refetchData } = useModels();
   const authorisedModels = sortBy([...collectionTypeLinks, ...singleTypeLinks], (model) =>
@@ -84,7 +85,7 @@ const App = () => {
   }

   return (
-    <Layout sideNav={<LeftMenu />}>
+    <Layout sideNav={isMobile() ? isLeftMenuOpen ? <LeftMenu /> : <></> : <LeftMenu />}>
       <DragLayer />
       <ModelsContext.Provider value={{ refetchData }}>
         <Switch>
@@ -114,7 +115,7 @@ const App = () => {

 export { App };

-export default function () {
+export default function (props) {
   const { formatMessage } = useIntl();

   return (
@@ -122,7 +123,7 @@ export default function () {
       <Helmet
         title={formatMessage({ id: getTrad('plugin.name'), defaultMessage: 'Content Manager' })}
       />
-      <App />
+      <App {...props} />
     </>
   );
 }
diff --git a/node_modules/@strapi/admin/admin/src/pages/Admin/index.js b/node_modules/@strapi/admin/admin/src/pages/Admin/index.js
index 13676c4..4089cd8 100644
--- a/node_modules/@strapi/admin/admin/src/pages/Admin/index.js
+++ b/node_modules/@strapi/admin/admin/src/pages/Admin/index.js
@@ -4,7 +4,7 @@
  *
  */

-import React, { Suspense, useEffect, useMemo, lazy } from 'react';
+import React, { Suspense, useEffect, useMemo, useState, lazy } from 'react';
 import { Switch, Route } from 'react-router-dom';
 import { useTracking, LoadingIndicatorPage, useStrapiApp } from '@strapi/helper-plugin';
 import { useDispatch, useSelector } from 'react-redux';
@@ -61,10 +61,13 @@ const useTrackUsage = () => {
   }, [appStatus]);
 };

+export const isMobile = () => window.innerWidth < 700
+
 const Admin = () => {
   useTrackUsage();
   const { isLoading, generalSectionLinks, pluginsSectionLinks } = useMenu();
   const { menu } = useStrapiApp();
+  const [isLeftMenuOpen, setIsLeftMenuOpen] = useState(!isMobile())

   const routes = useMemo(() => {
     return menu
@@ -83,6 +86,8 @@ const Admin = () => {
           <LeftMenu
             generalSectionLinks={generalSectionLinks}
             pluginsSectionLinks={pluginsSectionLinks}
+            initCondensedState={!isLeftMenuOpen}
+            onToggle={setIsLeftMenuOpen}
           />
         }
       >
@@ -90,7 +95,7 @@ const Admin = () => {
           <Switch>
             <Route path="/" component={HomePage} exact />
             <Route path="/me" component={ProfilePage} exact />
-            <Route path="/content-manager" component={CM} />
+            <Route path="/content-manager" ><CM isLeftMenuOpen={isLeftMenuOpen} /></Route>
             {routes}
             <Route path="/settings/:settingId" component={SettingsPage} />
             <Route path="/settings" component={SettingsPage} exact />
