diff --git a/node_modules/@strapi/plugin-email/server/controllers/email.js b/node_modules/@strapi/plugin-email/server/controllers/email.js
index 4ddb6ce..571a2a7 100644
--- a/node_modules/@strapi/plugin-email/server/controllers/email.js
+++ b/node_modules/@strapi/plugin-email/server/controllers/email.js
@@ -16,9 +16,9 @@ module.exports = {
       await strapi.plugin('email').service('email').send(options);
     } catch (e) {
       if (e.statusCode === 400) {
-        throw new ApplicationError(e.message);
+        throw new ApplicationError(e);
       } else {
-        throw new Error(`Couldn't send email: ${e.message}.`);
+        throw new Error(`Couldn't send email: ${e}.`);
       }
     }
 
@@ -45,9 +45,9 @@ module.exports = {
       await strapi.plugin('email').service('email').send(email);
     } catch (e) {
       if (e.statusCode === 400) {
-        throw new ApplicationError(e.message);
+        throw new ApplicationError(e);
       } else {
-        throw new Error(`Couldn't send test email: ${e.message}.`);
+        throw new Error(`Couldn't send test email: ${e}.`);
       }
     }
 
